GliaWidgets/ViewController/Call/CallViewController.swift

```diff
@@ -27,6 +27,11 @@ class CallViewController: EngagementViewController, MediaUpgradePresenter {
         view.willRotate(to: orientation, duration: duration)
     }
 
+    override func viewWillAppear(_ animated: Bool) {
+        guard let view = view as? CallView else { return }
```

Comment:
Is it expected to don't call `super` here?

Reply:
There was no intention of fully replacing `viewWillAppear`. Fixed, thanks for noticing.